Common/src/main/java/cz/neumimto/rpg/common/entity/players/CharacterService.java

```diff
@@ -664,7 +664,7 @@ public ActionResult canLearnSkill(T character, ClassDefinition classDef, ISkill
             return ActionResult.withErrorMessage(text);
         }
 
-        if (character.getLevel() < info.getMinPlayerLevel()) {
+        if (clazz.getLevel() < info.getMinPlayerLevel()) {
```

Comment:
Are these supposed to be z's?

Reply:
Yes,  'class' with double s is a reserved keyword in java